optimizer_wrapper.rb

```diff
@@ -585,6 +585,20 @@ def self.build_csv(solutions)
   private
 
   def self.check_result_consistency(expected_value, results)
+    [results].flatten(1).each{ |result|
+      waiting_time_map = result[:routes].map{ |route| route[:total_waiting_time] }.compact
+      duration_map = result[:routes].map{ |route|
```

Comment:
`x_map` has a specific meaning -- for example a mapping between variable ids and variables. here we have a simple array, I think we can just make do with pluralisation `waiting_times`, `durations`